src/fetchState/index.ts

```diff
@@ -0,0 +1,4 @@
+import fetchInitialState from './fetchInitialState'
+import fetchInitialStateAsync from './fetchInitialStateAsync'
+
+export { fetchInitialState, fetchInitialStateAsync }
```

Comment:
If you could use normal exports instead of default exports here, you could do something like this:
```suggestion
export * from './fetchInitialState'
export * from './fetchInitialStateAsync'
```

Reply:
I'm following here the rule of _default export if its only module export_ (as in `prefer-default-export` from eslint)

Reply:
It's a very controversial topic and more a design decision in my opinion. I really didn't liked it in my code that I have to implement more complexity for reexporting variables and that it is possible to introduce typos because default imports don't rely on the correct exported naming.

There is a huge discussion about it in this issue: https://github.com/airbnb/javascript/issues/1365

In my codebase I completely disabled this rule and going to use named exports only. Additionally it should help with tree shaking.